src/dialogs/ADialog.ts

```diff
@@ -24,42 +38,42 @@ abstract class ADialog {
 
   /**
    * creates a simple popup dialog under the given attachment
-   * @param attachment
-   * @param title
    * @param body
    * @returns {Selection<any>}
    */
   makePopup(body: string) {
-      const pos = offset(<Element>this.attachment.node());
-      const $popup = select('body').append('div')
-        .attr({
-          'class': 'lu-popup2'
-        }).style({
-          left: pos.left + 'px',
-          top: pos.top + 'px'
-        }).html(this.dialogForm(body));
-
-      function movePopup() {
-        //.style("left", (this.parentElement.offsetLeft + (<any>event).dx) + 'px')
-        //.style("top", (this.parentElement.offsetTop + event.dy) + 'px');
-        //const mouse = d3.mouse(this.parentElement);
-        $popup.style({
-          left: (this.parentElement.offsetLeft + (<any>d3event).dx) + 'px',
-          top: (this.parentElement.offsetTop + (<any>d3event).dy) + 'px'
-        });
-      }
+    const pos = offset(<Element>this.attachment.node());
+    const $popup = select('body').append('div')
+      .attr({
+        'class': 'lu-popup2'
+      }).style({
+        left: pos.left + 'px',
+        top: pos.top + 'px'
+      }).html(this.dialogForm(body));
 
-      $popup.select('span.lu-popup-title').call(behavior.drag().on('drag', movePopup));
-      $popup.on('keydown', () => {
-        if ((<KeyboardEvent>d3event).which === 27) {
-          $popup.remove();
-        }
+    function movePopup() {
+      //.style("left", (this.parentElement.offsetLeft + (<any>event).dx) + 'px')
+      //.style("top", (this.parentElement.offsetTop + event.dy) + 'px');
+      //const mouse = d3.mouse(this.parentElement);
+      $popup.style({
+        left: (this.parentElement.offsetLeft + (<any>d3event).dx) + 'px',
+        top: (this.parentElement.offsetTop + (<any>d3event).dy) + 'px'
       });
-      const auto = <HTMLInputElement>$popup.select('input[autofocus]').node();
-      if (auto) {
-        auto.focus();
+    }
+
+    $popup.select('span.lu-popup-title').call(behavior.drag().on('drag', movePopup));
+    $popup.on('keydown', () => {
+      if ((<KeyboardEvent>d3event).which === 27) {
+        $popup.remove();
       }
-      return $popup;
+    });
+    const auto = <HTMLInputElement>$popup.select('input[autofocus]').node();
+    if (auto) {
+      auto.focus();
```

Comment:
is this still needed that an element, which has the `autofocus` property, is saved to a variable and then the `focus` method is called?
Not sure if I introduced it, but I think it can be removed